keel-sql/src/main/kotlin/com/netflix/spinnaker/migrations/EventUidAssigner.kt

```diff
@@ -0,0 +1,73 @@
+package com.netflix.spinnaker.migrations
+
+import com.netflix.spinnaker.keel.persistence.metamodel.Tables.EVENT
+import com.netflix.spinnaker.keel.sql.inTransaction
+import de.huxhorn.sulky.ulid.ULID
+import java.time.LocalDateTime
+import java.time.ZoneOffset.UTC
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.jooq.DSLContext
+import org.jooq.Record1
+import org.slf4j.LoggerFactory
+import org.springframework.boot.context.event.ApplicationReadyEvent
+import org.springframework.context.event.EventListener
+
+class EventUidAssigner(private val jooq: DSLContext) : CoroutineScope {
```

Comment:
This will (slowly) plod through all the existing events and assign them a unique `uid`. We're not using that field anywhere (yet?) so I think it can harmlessly do this in the background. Once it's done them all (which by my calculations would take a couple of hours on a single instance) we can make `uid` a primary key column

Reply:
Seems like this PR comment should be a comment on the class itself.